.gitignore

```diff
@@ -1,3 +1,24 @@
-#Ignore files
-
-.DS_Store
+build/ 
+DerivedData/
+*.pbxuser 
+!default.pbxuser 
+*.mode1v3 
+!default.mode1v3 
+*.mode2v3 
+!default.mode2v3 
+*.perspectivev3 
+!default.perspectivev3 
+xcuserdata/
+*.moved-aside 
+*.xcuserstate
+*.hmap 
+*.ipa
+timeline.xctimeline 
+playground.xcworkspace
+.build/
+Pods/ 
+Carthage/Build
+fastlane/report.xml 
+fastlane/Preview.html 
+fastlane/screenshots
+fastlane/test_output
```

Comment:
Also you should add the files that get created when you run `pod install`
+ `app-steady.xcworkspace`
+ `Podfile.lock`